helm-charts/aikitcontainer/templates/imagestream.yaml

```diff
@@ -1,22 +1,86 @@
 apiVersion: image.openshift.io/v1
 kind: ImageStream
 metadata:
-  annotations:
-    opendatahub.io/notebook-image-name: "oneAPI AI Analytics Toolkit"
+  annotations: 
+    opendatahub.io/notebook-image-creator: 'kube:admin'
+    opendatahub.io/notebook-image-desc: ''
+    opendatahub.io/notebook-image-name: Intel® Optimized TensorFlow
+    opendatahub.io/notebook-image-url: 'quay.io/opendatahub/workbench-images:jupyter-intel-tensorflow-ubi9-python-3.9-{{ $.Values.opendatahub.commit }}'
+    opendatahub.io/recommended-accelerators: '["gpu.intel.com/i915"]'
   labels:
+    opendatahub.io/dashboard: 'true'
     opendatahub.io/notebook-image: "true"
-  name: {{ .Values.imagestream.name }}
-  namespace: {{ .Values.imagestream.namespace }}
+  name: intel-tensorflow
+  namespace: {{ $.Values.namespace }}
 spec:
   lookupPolicy:
     local: true
   tags:
-  - name: {{ .Values.imagestream.name }}
+  - name: intel-tensorflow
     annotations:
-      opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version":"3.4.4"},{"name":"Notebook","version":"7.2.2"}]'
-      opendatahub.io/notebook-software: '[{"name":"Python","version":"3.9"}, {"name":"AiKit","version":"2023.1.0"}]'
+      opendatahub.io/notebook-software: '[{"name":"Intel® oneAPI","version":"2024.0"},{"name":"Python","version":"v3.9"},{"name":"TensorFlow","version":"2.14"},{"name":"Intel® Extension for TensorFlow","version":"2.14"}]'
```

Comment:
Why not `python 3.10` since you are on `ubi9`?

Reply:
That's the current version of python in all notebook images in ODH.